.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+8e3d7f1a0114735005976434eb2b6baba9097bbd
```

Comment:
one last small request. lets include a comment that describes what the change was so that we know why that commit was ignored without needing to track it down in the actual history

Reply:
looks great!